apis/hive/v1/clusterpool_types.go

```diff
@@ -21,6 +21,12 @@ type ClusterPoolSpec struct {
 	// +required
 	Size int32 `json:"size"`
 
+	// RunningCount is the number of clusters we should keep running. The remainder will be kept hibernated until claimed.
+	// By default no clusters will be kept running (all will be hibernated).
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	RunningCount int32 `json:"runningCount"`
```

Comment:
can we have the json tag to include omitempty since this is optional field.

Reply:
Good catch, done.

(Though since this is a primitive type with a nil value, I suspect `omitempty` is redundant.)